wallet-new/src/Cardano/Wallet/API/V1/Wallets.hs

```diff
@@ -37,4 +37,13 @@ type API = Tags '["Wallets"] :>
                    :> Summary "Update the Wallet identified by the given walletId."
                    :> ReqBody '[ValidJSON] (Update Wallet)
                    :> Put '[ValidJSON] (WalletResponse Wallet)
+    :<|> "external-wallets"
+                   :> Summary "Creates a new or restores an existing external wallet (mobile client or hardware wallet)."
+                   :> ReqBody '[ValidJSON] (New ExternalWallet)
+                   :> PostCreated '[ValidJSON] (WalletResponse Wallet)
+    :<|> "external-wallets"
+                   :> CaptureWalletId
+                   :> "address-path"
+                   :> Summary "Generates a new BIP44 derivation path for the given external wallet."
+                   :> PostCreated '[ValidJSON] (WalletResponse AddressPath)
```

Comment:
same minor quibble re `wallets/external` instead of `external-wallets` for url path

Reply:
This is exactly what I thought, but Alfreado said that `external-wallets` is better...

Reply:
Let the wookie, er, manager win :trollface:

Reply:
Hah -- just applying the RESTful principles here 😉 

Apparently if you have something in your `/path`, that should map to a resource in the RESTful sense -- so if we have `/wallets/external` it means we are manipulating a thingy called `External`, which is not exactly very descriptive. Having `external-wallets` seemed more adhering to the best practices, but that's just my limited experience in the field, feel free to debunk 😉

Reply:
That makes sense, thanks for clarifying :)